src/main/java/software/amazon/awssdk/crt/checksums/Crc.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package software.amazon.awssdk.crt.checksums;
+
+import software.amazon.awssdk.crt.CrtResource;
+
+public abstract class Crc extends CrtResource {
```

Comment:
nitpicking maybe: should this class be called `Checksum` or `Checksums` then the methods being called crc32/crc32c make sense.

The "checksums" label redundancy between the namespace and class name is more common than having a redundancy in a class name and method name.